mangum/handlers/aws_alb.py

```diff
@@ -121,7 +121,7 @@ def request(self) -> Request:
             server_port = uq_headers.get("x-forwarded-port", 80)
         else:
             server_name, server_port = server_name.split(":")  # pragma: no cover
-        server = (server_name, int(server_port))
+        server = (server_name, int(server_port or 80))
```

Comment:
Since we already default to `80` when `:` is not present then we'd be re-casting the default value to `int` unnecessarily which is a bit confusing at a glance. 

Could you change this to instead remove the default returned from the headers lookup and set the default port conditionally based on the value of `server_port`? 

Maybe something like this:

```python
server_port = headers.get("x-forwarded-port")
if ":" in server_name:
    server_name, server_port = server_name.split(":")

server = (server_name, int(server_port) if server_port else 80)
```